pitch.md

```diff
@@ -1,60 +1,112 @@
-# Project Title (replace me)
+# Twitch Plays Programming
 
 ## Describe your project in five words
+Crowdsourcing videogame play via Twitch.
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+Since Twitch Plays Pokemon began in 2014, there has been lots of interest in
+creating similar "crowdplay" games. However, it is an area primarily done by
+people willing to overcome the steep curve of learning how to interface with
+emulators and Twitch's API. A DSL could help people who would have previously
+been excluded from making a "Twitch Plays".
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for multiple reasons. We are focusing on a specific domain,
+listening and executing Twitch chat commands. In addition, we are attempting to
+make that one domain extremely easy while allowing for everything else to be
+hard or impossible. We also want to allow the user to express potentially very
+complex compositions of commands, so the richness of a language is a big boost.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I was a big fan of the original Twitch Plays Pokemon and I was excited to see
+other people continue the tradition, but no one did. I attribute this significantly
+to the difficulty in creating such a project.
 
 ## Interface (syntax)
 _What kinds of things can the user say in your language. How, generally, do they
 say it? **Be as specific as possible.** You don't need to design syntax; but you
 should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
-important to users and what kinds of values those properties might have._ 
+important to users and what kinds of values those properties might have._
 
+The user would need to specify a couple of things. They would first need to specify
+their channel as the program needs to know which chat to scrape. They should also
+specify how often the game should execute its commands, e.g. every 10 votes or every
+5 seconds. The user might also want to specify how votes are counted, e.g. simple
+plurality or drawn from a hat. Finally, the user would need to specify what do when
+a command is chosen, e.g. when we choose "!jump", the spacebar should be pressed. I
+have ideas for other specifications, but these should be good for a minimum and I
+can extend them later.
 
```

Comment:
Would the DSL also encode ways to control the game players, or would it be merely deal with determining how to count the votes and choose what to do next?  If it is involved in game play, are there specific types of games that would be optimal for this DSL?  Could it communicate with other DSLs involved in executing the commands?